acs-apiclient/AcsApiClientConfig.cs

```diff
@@ -89,17 +91,26 @@ public class AcsApiClientConfig
         /// <summary>
         /// A token received upon completing the entire authentication flow
         /// </summary>
-        public string OAuthToken { get; set; }
+        internal string OAuthToken { get; set; }
 
         /// <summary>
         /// A secret received upon completing the entire authentication flow
         /// </summary>
-        public string OAuthSecret { get; set; }
+        internal string OAuthSecret { get; set; }
 
         /// <summary>
         /// Determines if the AcsApiClient was created from the SSO Flow
         /// </summary>
-        public bool IsSSOClient { get; internal set; } = false;
+        internal bool IsSSOClient { get; set; } = false;
+
+        /// <summary>
+        /// The expiration date of the OAuthToken.
+        /// </summary>
+        internal DateTime ExpirationDate { get; set; }
+
+        internal string EncodedTokenString => $"{ Utility.Base64Encode(OAuthToken) }" +
+            $".{ Utility.Base64Encode(OAuthSecret) }" +
+            $".{ Utility.Base64Encode(ExpirationDate.ToString("yyyy-MM-dd'T'HH:mm:ss'Z'")) }";
```

Comment:
Is this specific date format reused? If it is, maybe define this as a constant.

Reply:
👍

Reply:
:white_check_mark: